### PR TITLE
docs(theme): update navbar repo

### PR DESCRIPTION
**Pull request by xiaoyu2er**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
Please update en docs as well

**Comment by xiaoyu2er** [comment]
@Mister-Hope I updated the en docs

File: docs/theme/src/guide/layout/navbar.md [hunk 1]
```diff
@@ -538,12 +544,18 @@ const { defineHopeConfig } = require("vuepress-theme-hope");
...
     // Defaults to "GitHub" / "GitLab" / "Gitee" / "Bitbucket" or "Source" depending on `themeConfig.repo`
     repoLabel: "GitHub",
     // Whether to display repo link, default is `true`
     repoDisplay: true,
+    // If your docs are placed in a different repo, default is `repo`
+    docsRepo: 'vuepress-theme-hope/docs',
+    // branch where docs are located, default is `"main`
+    docsBranch: "main",
+    // dir whre docs are located, default is `""`
+    docsDir: "",
```
Comment (Mister-Hope) [inline docs/theme/src/guide/layout/navbar.md:558]:
```suggestion
    docsDir: "docs",
```

File: docs/theme/src/guide/layout/navbar.md [hunk 2]
```diff
@@ -517,12 +517,18 @@ import { defineHopeConfig } from "vuepress-theme-hope";
...
```
Comment (Mister-Hope) [inline docs/theme/src/guide/layout/navbar.md:531]:
```suggestion
    docsDir: "docs",
```

File: docs/theme/src/guide/layout/navbar.md [hunk 3]
```diff
@@ -517,12 +517,18 @@ import { defineHopeConfig } from "vuepress-theme-hope";
...
-    repo: "vuejs/vuepress",
+    repo: "vuepress-theme-hope/vuepress-theme-hope",
     // Customising the header label
     // Defaults to "GitHub" / "GitLab" / "Gitee" / "Bitbucket" or "Source" depending on `themeConfig.repo`
     repoLabel: "GitHub",
     // Whether to display repo link, default is `true`
     repoDisplay: true,
+    // If your docs are placed in a different repo, default is `repo`
+    docsRepo: 'vuepress-theme-hope/docs',
+    // branch where docs are located, default is `"main`
```
Comment (Mister-Hope) [inline docs/theme/src/guide/layout/navbar.md:528]:
```suggestion
    // Branch where docs are located, default is `"main`
```

File: docs/theme/src/guide/layout/navbar.md [hunk 4]
```diff
@@ -517,12 +517,18 @@ import { defineHopeConfig } from "vuepress-theme-hope";
...
+    docsBranch: "main",
+    // dir whre docs are located, default is `""`
```
Comment (Mister-Hope) [inline docs/theme/src/guide/layout/navbar.md:530]:
```suggestion
    // Dir whre docs are located, default is `""`
```

File: docs/theme/src/guide/layout/navbar.md [hunk 5]
```diff
@@ -538,12 +544,18 @@ const { defineHopeConfig } = require("vuepress-theme-hope");
...
```
Comment (Mister-Hope) [inline docs/theme/src/guide/layout/navbar.md:557]:
```suggestion
    // Dir whre docs are located, default is `""`
```

File: docs/theme/src/guide/layout/navbar.md [hunk 6]
```diff
@@ -538,12 +544,18 @@ const { defineHopeConfig } = require("vuepress-theme-hope");
...
-    repo: "vuejs/vuepress",
+    repo: "vuepress-theme-hope/vuepress-theme-hope",
     // Customising the header label
     // Defaults to "GitHub" / "GitLab" / "Gitee" / "Bitbucket" or "Source" depending on `themeConfig.repo`
     repoLabel: "GitHub",
     // Whether to display repo link, default is `true`
     repoDisplay: true,
+    // If your docs are placed in a different repo, default is `repo`
+    docsRepo: 'vuepress-theme-hope/docs',
+    // branch where docs are located, default is `"main`
```
Comment (Mister-Hope) [inline docs/theme/src/guide/layout/navbar.md:555]:
```suggestion
    // Branch where docs are located, default is `"main`
```

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

**Pull request merged**